addons/l10n_pl_jpk/models/account_move.py

```diff
@@ -0,0 +1,33 @@
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_pl_vat_b_spv = fields.Boolean(
+        string='B_SPV',
+        help="Transfer of a single-purpose voucher effected by a taxable person acting on his/its own behalf",
+        states={'draft': [('readonly', False)]},
+    )
+    l10n_pl_vat_b_spv_dostawa = fields.Boolean(
+        string='B_SPV_Dostawa',
+        help="Supply of goods and/or services covered by a single-purpose voucher to a taxpayer",
+        states={'draft': [('readonly', False)]},
+    )
+    l10n_pl_vat_b_mpv_prowizja = fields.Boolean(
+        string='B_MPV_Prowizja',
+        help="Supply of agency and other services pertaining to the transfer of a single-purpose voucher",
+        states={'draft': [('readonly', False)]},
+    )
```

Comment:
These are fields used to know if a voucher was used for it. It's not really related to taxes (and we don't have a specific account).
We could take care of it by indeed using a tax tag, but we would need to create a tax for it (while it makes no real sense for it to have a specific tax). 
I'm fine if we decide that's how we should do it, but that's not really logical for our users (and other softwares in Poland just put fields on the invoice).

Reply:
@gawa-odoo One more example for which I think we should have some kind of tagging mechanism directly on account.move ^^

@qdp-odoo We'll talk about this again for 18, I think ^^ We could also go for some kind of property object to store l10n-related fields, so that we don't "really" add columns to the tables for fields like that, but that would imply things in the ORM, so it'd be less easy.